draft-terbu-sd-jwt-vc.md

```diff
@@ -157,6 +157,39 @@ VC-SD-JWT with the following components:
 1. An SD-JWT to protect the integrity of the claims, to enable selective disclosure and to ensure authorship of the VC-SD-JWT. The SD-JWT header parameters and the payload is further defined by this specification.
 1. The full set of SD-JWT Disclosures that contain the claims in plain text.
 
+# Flow Diagram
+~~~ ascii-art
+           +------------+
+           |            |
+           |   Issuer   |
+           |            |
+           +------------+
+                 |
+             Issues SD-JWT
+     and Issuer-Issued Disclosures
+                 |
+                 v
+           +------------+                                 +------------+ 
+           |            |                                 |   Status   |
+           |   Holder   |-------- optionally ------------>|  Provider  |
+           |            |       retrieves status          |            |
+           +------------+             VC                  +------------+ 
+                 |
+           Presents SD-JWT
+   and Holder-Selected Disclosures
+                 |
```

Comment:
Thanks for the PR. can you change the terms used for the labels of the lines? Let's avoid SD-JWT and use Verifiable Credentials and Verifiable Presentations instead.

Reply:
Done in [cad5733](https://github.com/vcstuff/draft-terbu-sd-jwt-vc/pull/55/commits/cad5733884ac2c61748350a314965f95e8a5d749)